lib/gltf.js

```diff
@@ -154,185 +43,301 @@ function createGltf(data, inputPath, modelName) {
     };
 
     gltf.asset = {
-        "generator": "OBJ2GLTF",
-        "premultipliedAlpha": true,
-        "profile": {
-            "api": "WebGL",
-            "version": "1.0"
+        generator : 'obj2gltf',
+        profile : {
+            api : 'WebGL',
+            version : '1.0'
         },
-        "version": 1
+        version: '1.0'
     };
 
     gltf.scenes[sceneId] = {
-        nodes : [nodeId]
+        nodes : []
     };
 
-    gltf.nodes[nodeId] = {
-        children : [],
-        matrix : [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1],
-        meshes : [meshId],
-        name : modelName
-    };
+    function getImageId(imagePath) {
```

Comment:
I think this crossed my mind too, but it seemed too unlikely to happen. Of course I will probably get proven wrong at some point.